CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#739](https://github.com/nix-rust/nix/pull/739))
 - Expose `signalfd` module on Android as well.
   ([#739](https://github.com/nix-rust/nix/pull/739))
+- Added enums for ptrace request, event and options to replace constants. 
```

Comment:
This should be under "Changed", since we changed a public API, we didn't add a new one really. I would say something like "Changed the `PtraceEvent`, `PtraceOptions`, and `PtraceRequest` datatypes into enums and updated ptrace function argument types accordingly."